css/style.css

```diff
@@ -96,3 +96,399 @@ nav {
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.02em;
+}
+
+.about {
+  width: 375px;
+  height: 955px;
+  left: 375px;
+  top: 4769px;
+  background: white;
+  border-radius: 0 88px 0 0;
+}
+
+.js,
+.html,
+.css {
+  display: flex;
+  width: 303px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+}
+
+.jstext,
+.csstext,
+.htmltext {
+  margin: 0 0 0 39px;
+}
+
+.jstext > p,
+.csstext > p,
+.htmltext > p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.skillz {
+  display: flex;
+  justify-content: space-between;
+}
+
+.chevron > img {
+  margin: 20px 30px 0 0;
+}
+
+.contactme {
+  background-image: url(../img/contact.png);
+  padding-top: 2.5em;
+}
+
+.contactme h2 {
+  top: 4734px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.contacttext {
+  width: 327px;
+  height: 102px;
+  top: 4802px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin: 0 0 0 20px;
+}
+
+.contactform {
+  padding-top: 1em;
+  background-image: url(../img/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+}
+
+.contactform > form {
+  padding-top: 82px;
+}
+
+#submit {
+  padding: 12px 16px;
+  width: 140px;
+  background: #fff;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  margin: 4px 0 0 11px;
+  border: none;
+}
+
+#textarea {
+  margin: 0 0 1px 13px;
+  width: 336px;
+  height: 117px;
+  border-radius: 9px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+}
+
+.programming {
+  display: flex;
+  justify-content: space-between;
+}
+
+.programming img {
+  margin: 3px;
+  margin: 19px 19px 0 0;
+}
+
+.programming p {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+#name {
+  margin: 0 0 1px 13px;
+  width: 336px;
+  height: 33px;
+  border-radius: 9px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+}
+
+#email {
+  margin: 0 0 1px 13px;
+  width: 336px;
+  height: 33px;
+  border-radius: 9px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+}
+
+#email::placeholder,
+#name::placeholder {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+.cardbut {
+  box-sizing: border-box;
+  flex-direction: column;
```

Comment:
- [ ] Kindly implement the buttons styling for hover, active and pressed. 👍 

## Figma Design Template
 
![crr8](https://user-images.githubusercontent.com/103386183/198285216-c09f63dd-e974-430b-b64e-f2e958629b8c.png)

- [ ] Also, all your buttons are not properly aligned with other content; you may have to adjust the margin-left. 👍 


![crr9](https://user-images.githubusercontent.com/103386183/198286252-01bc99c3-9bc5-4a03-a377-dc69796b25d5.png)

Reply:
## Resolved 👍 
- added the effects
- effects added
    - on hover
    - on press
    - active
- thanks for noticing

![Jack Sparrow Salute](https://media.giphy.com/media/2PmMiUc9OH4lO/giphy.gif)